fixtures/fixtures.js

```diff
@@ -43,6 +43,18 @@ var Fixtures = {
             {"code": "SOL00000012", "updated_at": "17-05-2017 14:46.15"},
         ]
     },
+    vouchersStatus: {
```

Comment:
These aren't being used, are they?. I think you need to set up as I described below so it matches what the api does - rather than being a list of voucher objects. Should be the status returned, not the vouchers.